CUBRIDMAN-222 :매뉴얼 수정 작업

bagus-kim opened the pull request:
CUBRIDMAN-222 : RND-2001 : 볼륨 호환성 표 추가
CUBRIDMAN-222 : RND-1591 : max_plan_cache_clones,xasl_cache_time_threshold_in_minutes설명 및 예문 추가
CUBRIDMAN-222 : RND-2251 : 동시성 설명 중, 유령읽기 관련 수정
CUBRIDMAN-222 : RND-2253 : cubrid heartbeat status -v의 설명 및 예문추가
CUBRIDMAN-222 : RND-2283 : index.html에 라이선스 가이드 URL 수정 작업

kwonhoil commented on en/upgrade.rst:
```diff
@@ -44,6 +44,47 @@ Please confirm :ref:`11_3_changes` in the release notes.
+| 8.2.0     | X   | X   | X   | X   | X   | X   | X  | X  | X    | X    | X    | X    | X    | X    | X    | O    | O    |
++-----------+-----+-----+-----+-----+-----+-----+----+----+------+------+------+------+------+------+------+------+------+
+
+
```
X, O 표기가 함께 있어서 가독성이 많이 떨어집니다.

O만 표기하는게 좋을것 같습니다.

bagus-kim replied:
의견주셔서 감사합니다. 반영했습니다.

kwonhoil commented on ko/upgrade.rst:
```diff
@@ -44,6 +44,46 @@
++-----------+-----+-----+-----+-----+-----+-----+----+----+------+------+------+------+------+------+------+------+------+
+| 8.2.0     | X   | X   | X   | X   | X   | X   | X  | X  | X    | X    | X    | X    | X    | X    | X    | O    | O    |
++-----------+-----+-----+-----+-----+-----+-----+----+----+------+------+------+------+------+------+------+------+------+
+
```
X, O 표기가 함께 있어서 가독성이 많이 떨어집니다.

O만 표기하는게 좋을것 같습니다.

mhoh3963 replied:
매뉴얼에서는 EOL된 제품인 9.3 이하 버전에 대해서는 표시하지 않는 것이 어떨까요 ?

bagus-kim replied:
의견주셔서 감사합니다. 반영했습니다.

bagus-kim committed:
CUBRIDMAN-222 : RND-2001 : 볼륨 호환성 표 추가
CUBRIDMAN-222 : RND-1591 : max_plan_cache_clones,xasl_cache_time_threshold_in_minutes설명 및 예문 추가
CUBRIDMAN-222 : RND-2251 : 동시성 설명 중, 유령읽기 관련 수정
CUBRIDMAN-222 : RND-2253 : cubrid heartbeat status -v의 설명 및 예문추가

mhoh3963 commented on ko/ha.rst:
```diff
@@ -1394,6 +1394,57 @@ CUBRID HA 그룹 정보와 CUBRID HA 구성 요소의 정보를 확인할 수 
+
+* registered-time : 명령어를 통하여 프로세스 구동 요청 시간
+* deregistered-time : 명령어를 통하여 원격 프로세스 정지 요청 시간 (copylogdb와 applylogdb만 해당)
+* shutdown-time : 프로세스 정지시간
```
```suggestion
* shutdown-time : HA 매니저(cub_master)가 프로세스를 정지한 시간
```

bagus-kim replied:
반영하였습니다.

mhoh3963 commented on ko/ha.rst:
```diff
@@ -1394,6 +1394,57 @@ CUBRID HA 그룹 정보와 CUBRID HA 구성 요소의 정보를 확인할 수 
+* registered-time : 명령어를 통하여 프로세스 구동 요청 시간
+* deregistered-time : 명령어를 통하여 원격 프로세스 정지 요청 시간 (copylogdb와 applylogdb만 해당)
+* shutdown-time : 프로세스 정지시간
+* start-time : 프로세스 재 구동시간 
```
```suggestion
* start-time : HA 매니저(cub_master)가 프로세스를 재 구동시간 
```

bagus-kim replied:
반영하였습니다.

mhoh3963 commented on ko/admin/config.rst:
```diff
@@ -1991,6 +1995,12 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
         max_plan_cache_entries=1000
 
+**max_plan_cache_clones**
+    plan cache에 query에 대한 xasl이 serializable하게 되어 있는데, query 실행시에는 다시 xasl구조로 변경(deserializable)해서 value를 채워 수행한다. clone cache은 deserializable한 xasl을 버리지 않고 재 사용하기 위한 cache 파라미터 이다. 기본갑은 1000이며, 하나의 plan이 최대로 가질 수 있는 cache값입니다. plan이 삭제되면clone도 함께 삭제된다. 
```
```suggestion
    플랜 캐시(plan cache)에는 질의에 대한 직렬화(serializable)된 xasl 저장되어 있는데, 질의 수행기는 플랜 캐시에 저장된 작렬화된 xasl를 역직렬화(deserializable)한 후 사용할 값을 채운 후 수행한다. 클론 캐시(clone cache)는 질의 수행기가  최소한의 역직렬화 수행하기 위해 역직렬화된 xasl을 저장하여 재 사용하기 위한 캐시 파라미터이다. 기본값은 1000이며, 하나의 플랜이 최대로 가질 수 있는 캐시 값입니다. 플랜이 삭제되면 해당 플랜의 클론 캐시에 저장된 역직렬화된 xasl도 함께 삭제된다. 
```

bagus-kim replied:
리뷰 감사합니다. 반영하였습니다.

mhoh3963 commented on ko/admin/config.rst:
```diff
@@ -1991,6 +1995,12 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
+    plan cache에 query에 대한 xasl이 serializable하게 되어 있는데, query 실행시에는 다시 xasl구조로 변경(deserializable)해서 value를 채워 수행한다. clone cache은 deserializable한 xasl을 버리지 않고 재 사용하기 위한 cache 파라미터 이다. 기본갑은 1000이며, 하나의 plan이 최대로 가질 수 있는 cache값입니다. plan이 삭제되면clone도 함께 삭제된다. 
+
+**xasl_cache_time_threshold_in_minutes**
+     cache된 plan을 재 수행(clean-up)할 것인지 판단하는 시간으로, plan cache에 추가할 공간이 없는 경우에도 제거할 대상을 찾는 부분에서도 사용한다. 분단위로 설정 가능하며, 기본값은 360분이다.
```
```suggestion
     캐시된 플랜을 재 수행(clean-up)할 것인지를 판단하는 파라미터로, 해당 파라미터는 플랜 캐시에 추가할 공간이 없는 경우 제거할 대상을 찾기 위해 사용된다. 분단위로 설정 가능하며, 기본값은 360분이다.
```

bagus-kim replied:
리뷰 감사합니다. 반영하였습니다.

mhoh3963 commented on en/admin/config.rst:
```diff
@@ -2019,6 +2023,13 @@ The following are parameters related to the query plan cache functionality. The
+     The clone cache stores deserialized XASL to reuse it instead of discarding. The **max_plan_cache_clones** is a parameter that sets the maximum number of cloned cache entries a single plan can hold, and its default value is set to 1000. When a plan is deleted, its clones are also deleted.
+
+**xasl_cache_time_threshold_in_minutes**
+     **xasl_cache_time_threshold_in_minutes** is a parameter that determines the time threshold for deciding whether to reuse (clean-up) a cached plan. It is also used when searching for a candidate plan to remove when there is no space in the plan cache. This parameter can be set in minutes, with a default value of 360 minutes.
```
```suggestion
     It is a parameter that determines the time threshold for deciding whether to recompile (clean-up) a cached plan. It is also used when searching for a candidate plan to remove when there is no space in the plan cache. This parameter can be set in minutes, with a default value of 360 minutes.
```

bagus-kim replied:
리뷰 감사합니다. 반영하였습니다.

mhoh3963 commented on ko/admin/config.rst:
```diff
@@ -1991,6 +1995,12 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
         max_plan_cache_entries=1000
 
+**max_plan_cache_clones**
+    플랜 캐시(plan cache)에는 질의에 대한 직렬화(serializable)된 xasl 저장되어 있는데, 질의 수행기는 플랜 캐시에 저장된 작렬화된 xasl를 역직렬화(deserializable)한 후 사용할 값을 채운 후 수행한다. 클론 캐시(clone cache)는 질의 수행기가  최소한의 역직렬화 수행하기 위해 역직렬화된 xasl을 저장하여 재 사용하기 위한 캐시 파라미터이다. 기본값은 1000이며, 하나의 플랜이 최대로 가질 수 있는 캐시 값입니다. 플랜이 삭제되면 해당 플랜의 클론 캐시에 저장된 역직렬화된 xasl도 함께 삭제된다. 
```
```suggestion
    플랜 캐시(plan cache)에는 질의에 대한 직렬화(serializable)된 xasl 저장되어 있는데, 질의 수행기는 플랜 캐시에 저장된 작렬화된 xasl를 역직렬화(deserializable)한 후 사용할 값과 함께 수행한다. 클론 캐시(clone cache)는 질의 수행기가  최소한의 역직렬화 수행하기 위해 역직렬화된 xasl을 저장하여 재 사용하기 위한 캐시 파라미터이다. 기본값은 1000이며, 하나의 플랜이 최대로 가질 수 있는 캐시 값입니다. 플랜이 삭제되면 해당 플랜의 클론 캐시에 저장된 역직렬화된 xasl도 함께 삭제된다. 
```

bagus-kim replied:
반영하였습니다

mhoh3963 commented on ko/admin/config.rst:
```diff
@@ -1991,6 +1995,12 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
+    플랜 캐시(plan cache)에는 질의에 대한 직렬화(serializable)된 xasl 저장되어 있는데, 질의 수행기는 플랜 캐시에 저장된 작렬화된 xasl를 역직렬화(deserializable)한 후 사용할 값을 채운 후 수행한다. 클론 캐시(clone cache)는 질의 수행기가  최소한의 역직렬화 수행하기 위해 역직렬화된 xasl을 저장하여 재 사용하기 위한 캐시 파라미터이다. 기본값은 1000이며, 하나의 플랜이 최대로 가질 수 있는 캐시 값입니다. 플랜이 삭제되면 해당 플랜의 클론 캐시에 저장된 역직렬화된 xasl도 함께 삭제된다. 
+
+**xasl_cache_time_threshold_in_minutes**
+     캐시된 플랜을 재 수행(clean-up)할 것인지를 판단하는 파라미터로, 해당 파라미터는 플랜 캐시에 추가할 공간이 없는 경우 제거할 대상을 찾기 위해 사용된다. 분단위로 설정 가능하며, 기본값은 360분이다.
```
```suggestion
     캐시된 플랜을 재 수행(clean-up)할 것인지를 판단하는 파라미터로, 해당 파라미터는 플랜 캐시에 추가할 공간이 없는 경우도 제거할 대상을 찾기 위해 사용된다. 분단위로 설정 가능하며, 기본값은 360분이다.
```

bagus-kim replied:
반영하였습니다.

bagus-kim committed:
CUBRIDMAN-222 : upgrade.rst, ha.rst에 대한 리뷰 반영

bagus-kim committed:
CUBRIDMAN-222 : config.rst에 대한 리뷰 반영

bagus-kim committed:
CUBRIDMAN-222 : admin/config.rst에 대한 리뷰 반영

bagus-kim committed:
[CUBRIDMAN-222] : RND-2283 라이선스 링크 오류 수정
